ERCS/erc-7779.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 7779
+title: Interoperable Delegated Accounts
+description: Interoperable Delegated Account interfaces and behaviors for better interoperability and storage management.
```

Comment:
Your description doesn't do a good job of introducing new information that wasn't already in your title. I'd recommend using this field to explain what an "interoperable delegated account" actually is.

Reply:
I updated the description section to be more inline with the title